model/pipeline.js

```diff
@@ -79,5 +79,21 @@ module.exports = {
      * @property keys
      * @type {Array}
      */
-    keys: ['scmUrl']
+    keys: ['scmUrl'],
+
+    /**
+     * Tablename to be used in the datastore
+     *
+     * @property tableName
+     * @type {String}
+     */
+    tableName: 'pipelines',
```

Comment:
Are we going with `Pipelines` or `pipelines`? (https://github.com/screwdriver-cd/dynamic-dynamodb/pull/1/files#diff-64fbbb5a69337f996786fef9c19e1271R23)